client/components/users/VerifyCertificateNum.js

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from "react";
+import "./VerifyCertificateNum.css";
+import { toast, ToastContainer } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
+import axios from "axios";
+
+export function VerifyCertificateNum() {
+	const [email, setEmail] = useState("");
+	const [certificateNum, setCertificateNum] = useState("");
+	const [loading, setLoading] = useState(false);
+
+	const handleSubmit = async (event) => {
+		event.preventDefault();
+		if (!email || !certificateNum) {
+			return toast.error("Please provide email and certificate Number");
+		}
+		try {
+			setLoading(true);
+			const { data } = await axios.post(`/api`, {
+				email,
+				certificateNum,
+			});
+			setLoading(false);
+			toast.success(data.message);
+		} catch (err) {
+			setLoading(false);
+			toast.error(
+				err.response && err.response.data.message
+					? err.response.data.message
+					: err.message
+			);
+		}
+	};
+
+	return (
+		<div className="signUpInfoContainer">
+			<ToastContainer position="bottom-center" limit={1} />
+			<h2 className="signUpHeader"> Sign Up As A Graduate </h2>
+			<form onSubmit={handleSubmit}>
+				<div>
+					<label for="email"> Email</label>
```

Comment:
This should be `<label htmlFor="email"> Email</label>`

Reply:
Hi, I have amended it. thanks.